File: Auto-generated-tasks/Difficulty-1/Task-1/Solution.cs
int NN = 5;
int[] Data = new int[NN];
int u, A1 = 0;
for (u = 0; u < NN; u++) {
	Data[u] = u * 9;
	A1 = A1 + Data[u];
	Console.WriteLine($"Data[{u}]={Data[u]}");
}
Console.WriteLine("A1=" + A1);

File: Auto-generated-tasks/Difficulty-1/Task-1/Solution.php
<?
$NN = 5;
$Data = array_fill(0, $NN, 0);
$u = 0;
$A1 = 0;
for ($u = 0; $u < $NN; $u++) {
	$Data[$u] = $u * 9;
	$A1 = $A1 + $Data[$u];
	print("Data[", $u, "]=", $Data[$u]);
}
print("R=" + $A1);
?>